src/vscode-bicep/src/commands/build.ts

```diff
@@ -18,6 +18,14 @@ export class BuildCommand implements Command {
       return;
     }
 
+    if (documentUri.scheme === "output") {
+      vscode.window.showInformationMessage(
+        "We are unable to get the Bicep file to build when the output panel is focused. Please focus a text editor first when running the command."
+      );
```

Comment:
This is actually a bug (I guess now it's a feature) in VS Code. The output panel was implemented as a text editor by accident long time ago, but due to breaking change concerns it won't be fixed in VS Code (see https://github.com/microsoft/vscode/issues/58869#issuecomment-422322972).

Reply:
Good to know! Would you mind also adding this as a comment to the source?

Reply:
Good pt! Added notes to the source.